### PR TITLE
Add boxes BSC and Material-UI

### PR DESCRIPTION
**Pull request by leeftk**
This PR adds the [BSC](https://github.com/RumeelHussainbnb/BSC-Truffle-Starter-Box) and [React-Material-UI](https://github.com/rouftom/truffle-react-material) boxes to the Truffle website as community boxes.

### CODE REVIEW

File: custom_theme/boxes.html [hunk 1]
```diff
@@ -52,10 +52,16 @@ <h2>The easiest way to get started</h2>
       <div class="col-xs-12 col-md-6 col-lg-4">
           <a href="{{ box.displayName }}/">
             <div class="card">
+<<<<<<< HEAD
```
Comment (kevinbluer) [inline custom_theme/boxes.html:55]: hey @leeftk looks like there's a bit of an issue with the merging here (and below)

**Review by kevinbluer — CHANGES_REQUESTED** [review]
Just the merging issue (as highlighted)...otherwise looks good

File: custom_theme/boxes.html (hunk 1 above)
Comment (leeftk) [reply custom_theme/boxes.html]: Rebased and fixed these!

File: src/data/boxes.json [hunk 2]
```diff
@@ -14,13 +14,31 @@
...
+	},
 	"angular-truffle-box": {
 		"description": "Truffle Box for Angular is a quick-and-easy way to get your Dapp on the road with Truffle and Angular",
 		"stars": 49,
 		"tags": [
 			"community"
 		]
 	},
+	"truffle-react-material": {
+		"description": "This box contains everything you need to get started using smart contracts from a Rect app with the Material-ui library.",
```
Comment (kevinbluer) [inline src/data/boxes.json:34]: Typo on "Rect"

**Review by kevinbluer — CHANGES_REQUESTED** [review]
Just a typo, otherwise looks good to go 👌

File: src/data/boxes.json (hunk 2 above)
Comment (leeftk) [reply src/data/boxes.json]: Nice catch!

**Review by kevinbluer — APPROVED** [review]
Good to go 👌

**Pull request merged**